src/Command/Model/DtoTemplate/ResponseDtoTemplate.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Centreon\Command\Model\DtoTemplate;
+
+use Centreon\Command\Model\FileTemplate;
+
+class ResponseDtoTemplate extends FileTemplate
+{
+    public function __construct(
+        public string $filePath,
+        public string $namespace,
+        public string $name,
+        public bool $exists = false
+    ) {
+        parent::__construct();
+    }
+
+    public function generateModelContent(): string
+    {
+        $content = <<<EOF
+        <?php
+        $this->licenceHeader
+        declare(strict_types=1);
+
+        namespace $this->namespace;
+
+        class $this->name
+        {
+        }
+
+        EOF;
+
+        return $content;
+    }
+
+    public function __toString()
```

Comment:
Please use the Stringable interface in the definition of the class and specify the return type.

Reply:
Done